<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OpenRGBServerInfoPage</class>
 <widget class="QFrame" name="OpenRGBServerInfoPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>664</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true">Server Info Page</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="2" column="3">
    <widget class="QPushButton" name="ServerStopButton">
     <property name="text">
      <string>Stop Server</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="ServerPortLabel">
     <property name="text">
      <string>Server Port:</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="ServerClientsLabel">
     <property name="text">
      <string>Connected Clients:</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0" colspan="4">
    <widget class="QTreeWidget" name="ServerClientTree">
     <property name="columnCount">
      <number>3</number>
     </property>
     <column>
      <property name="text">
       <string>Client IP</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Protocol Version</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Client Name</string>
      </property>
     </column>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLineEdit" name="ServerPortValue">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="ServerStatusLabel">
     <property name="text">
      <string>Server Status:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="3">
    <widget class="QPushButton" name="ServerStartButton">
     <property name="text">
      <string>Start Server</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLabel" name="ServerStatusValue">
     <property name="text">
      <string>Offline</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="ServerHostLabel">
     <property name="text">
      <string>Server Host:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="ServerHostValue">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>ServerPortValue</tabstop>
  <tabstop>ServerStartButton</tabstop>
  <tabstop>ServerStopButton</tabstop>
  <tabstop>ServerClientTree</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
